### ISSUE TITLE
after reconnecting, store does not resume sync

### ISSUE THREAD

**Issue by gr2m** [hoodiehq/hoodie-client#133]

**Comment by Alice-anjali** [comment]
Hi I went through the code as well as the APIs of this issue. As you mentioned above that the event is not called 'connect' but 'reconnect' and there are only 3 events declared in the hoodie-connection-status which are 'disconnect, reconnect and reset'. There is no event named 'connect'. So I guess if we just change 'connect' to 'reconnect', would it solve the issue ?

**Comment by gr2m** [comment]
@Alice-anjali yes, I think so. It might break some tests so these will need to be fixed, too. And if the change does not break any tests, then we should probably add a test for it :) Do you want to give a try?

**Comment by Alice-anjali** [comment]
Yes I would definitely give it a try! ^_^

**Comment by Alice-anjali** [comment]
I changed 'connect' to 'reconnect' and checked it. There are no tests breaking. @sohini-roy is already writing related test cases in the coverage issue. So, can I make a PR for this?

**Comment by gr2m** [comment]
> can I make a PR for this?

Yes, please :) If you run into any blockers with the tests, let’s discuss there

**Comment by Alice-anjali** [comment]
I made a PR, please review it. :-)
